schutzbot/Jenkinsfile

```diff
@@ -165,6 +165,69 @@ pipeline {
                         }
                     }
                 }
+                stage('F33 Base') {
+                    agent { label "f33cloudbase && x86_64 && aws" }
+                    environment { TEST_TYPE = "base" }
+                    steps {
+                        unstash 'fedora33'
+                        run_tests('base')
+                    }
+                    post {
+                        always {
+                            preserve_logs('fedora33-base')
+                        }
+                    }
+                }
+                stage('F33 Image') {
+                    agent { label "f33cloudbase && psi && x86_64" }
+                    environment {
+                        TEST_TYPE = "image"
+                        AWS_CREDS = credentials('aws-credentials-osbuildci')
+                        AWS_IMAGE_TEST_CREDS = credentials('aws-credentials-osbuild-image-test')
+                        AZURE_CREDS = credentials('azure')
+                        OPENSTACK_CREDS = credentials("psi-openstack-creds")
+                        VCENTER_CREDS = credentials('vmware-vcenter-credentials')
+                        DISTRO_CODE = "fedora33"
+                    }
+                    steps {
+                        unstash 'fedora33'
+                        run_tests('image')
+                    }
+                    post {
+                        always {
+                            preserve_logs('fedora33-image')
```

Comment:
```suggestion
                            preserve_logs('fedora33-image')
                            sh (
                                label: "Run cloud cleaner just in case something failed",
                                script: "schutzbot/run_cloud_cleaner.sh"
                            )
```

Reply:
Indeed, thanks! I added those lines before the cloud cleaner was introduced.